README.md

```diff
@@ -171,14 +175,17 @@ export default connect( mapStateToProps )( App );
 Now that our application is talking to Redux we need to set up Redux to actually do the things we want it to do. We'll start by creating action types, creating action creators, and implementing increment/decrement logic.
 
 **Instructions**
-
+<!-- This comma series is hard to read. Splitting into three sentences could clear things up. -->
```

Comment:
Switching all of these to bullets per your comment. I tend to over-bullet, now I guess I'm overcompensating in the other direction. 🥇